src/components/AttachResult/AttachResult.ts

```diff
@@ -194,8 +203,11 @@ export class AttachResult extends Component {
         let customData: IAnalyticsCaseAttachMeta = {
             resultUriHash: this.queryResult.raw.urihash,
             author: this.queryResult.raw.author,
-            articleID: null,
-            caseID: null
+            articleID:
+                this.options.articleIdField && this.queryResult.raw[this.options.articleIdField]
+                    ? this.queryResult.raw[this.options.articleIdField]
+                    : null,
```

Comment:
Can be simplified like so:
`articleID: this.queryResult.raw[this.options.articleIdField] ?? null`
I think.